Sources/TuistGenerator/Extensions/Graph+Extras.swift

```diff
@@ -21,6 +22,10 @@ extension TuistGraph.Graph {
                 return false
             }
 
+            if let platformToFilter = platformToFilter, target.target.platform != platformToFilter {
```

Comment:
```suggestion
            if target.target.platform != platformToFilter {
```

Reply:
When `platformToFilter` is `nil`(the default value), nothing goes through that filter and we want the exact opposite behavior (everything goes through).

Reply:
right, sorry :D
I would still use a guard as it's a precondition for going on:
```
guard platformToFilter == nil || target.target.platform == platformToFilter else {
```